libs/template/templates/default-sql/databricks_template_schema.json

```diff
@@ -0,0 +1,53 @@
+{
+    "welcome_message": "\nWelcome to the default SQL template for Databricks Asset Bundles!",
+    "properties": {
+        "project_name": {
+            "type": "string",
+            "default": "sql_project",
+            "description": "\nPlease provide a unique name for this project.\nproject_name",
+            "order": 1,
+            "pattern": "^[A-Za-z0-9_]+$",
+            "pattern_match_failure_message": "Name must consist of letters, numbers, and underscores."
+        },
+        "http_path": {
+            "type": "string",
+            "pattern": "^/sql/.\\../warehouses/[a-z0-9]+$",
+            "pattern_match_failure_message": "Path must be of the form /sql/1.0/warehouses/abcdef1234567890",
```

Comment:
Ideally there is some indication in the failure message path which parts of the path are expected text and which parts are placeholders.